api/src/main/java/org/openmrs/obs/handler/AbstractHandler.java

```diff
@@ -56,49 +59,95 @@ public AbstractHandler() {
 	 * and the file name is determined off the current obs.getComplexData().getTitle().
 	 * 
 	 * @param obs the Obs with a non-null complex data on it
-	 * @return File that the complex data should be written to
+	 * @return file that the complex data should be written to
 	 */
 	public File getOutputFileToWrite(Obs obs) throws IOException {
-		// Get the title and remove the extension.
-		String t = obs.getComplexData().getTitle();
```

Comment:
Could you remind me why we removed these two lines?

Reply:
I just renamed the variable to title, it's some lines down, to make it easier to read. At least that's how it looks like, it's been some time.